### ISSUE TITLE
Acts like Javascript is disabled when it's enabled

### ISSUE THREAD

**Issue by cmshawns** [resume/resume.github.com#195]
In both Firefox & Chrome (on Windows) I get the following page body:

```html
<body id="resume">
	<noscript>You need to enable Javascript to generate and see r&eacute;sum&eacute; pages.</noscript>
</body>
```

I do have Javascript-disabling extensions installed (NoScript on Firefox and ScriptSafe on Chrome), but Javascript has been enabled for both the resume.github.com and resume.github.io - as evidenced by the fact that I don't see the `<noscript>` message when viewing the page.

**Comment by winniehell** [comment]
This was the issue for me: https://blog.mozilla.org/tanvi/2013/04/10/mixed-content-blocking-enabled-in-firefox-23/

**Comment by LB--** [comment]
It looks like it is caused by viewing via HTTPS instead of HTTP.

**Comment by winniehell** [comment]
@LB-- That sounds like it is reprehensible to use encrypted connections... :smile:

**Comment by u2** [comment]
It has been fixed #182
